### PR TITLE
TIB 1.6 release notes

### PR DESCRIPTION
**Pull request by carolwkc**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<br>

https://deploy-preview-5694--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-identity-broker/release-notes/tib-v1.6/

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

___

### **PR Type**
Documentation

___

### **Description**
- Added comprehensive release notes for Tyk Identity Broker version 1.6.x, detailing updates, enhancements, and changes.
- Introduced support for JSON Web Encryption (JWE) in OIDC Single Sign-On (SSO) for enhanced security.
- Updated compatibility information for third-party dependencies such as GoLang, MongoDB, PostgreSQL, and Redis.
- Provided detailed upgrade instructions and download links for the new release.
- Updated the documentation menu to include the new release notes for easy navigation.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>tib-v1.6.md</strong><dd><code>Add release notes for Tyk Identity Broker v1.6.x</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-identity-broker/release-notes/tib-v1.6.md

<li>Added release notes for Tyk Identity Broker version 1.6.x.<br> <li> Documented support for JSON Web Encryption (JWE) in OIDC SSO.<br> <li> Included compatibility details for third-party dependencies.<br> <li> Provided upgrade instructions and download links.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5694/files#diff-c8f45d24b9f8ca5c53776e9a96c8cd6dfe7e62941c853d2975e46e5e3d9b31cb">+162/-0</a>&nbsp; </td>

</tr>                    
</table></td></tr><tr><td><strong>Configuration changes</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>menu.yaml</strong><dd><code>Update menu to include Tyk Identity Broker 1.6 release notes</code></dd></summary>
<hr>

tyk-docs/data/menu.yaml

<li>Added menu entries for Tyk Identity Broker 1.6 release notes.<br> <li> Updated navigation structure to include new release notes.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5694/files#diff-a6789c66bfa0660f7fdbe4796002ab4777677bae59a0276689c840100ff0b0b2">+8/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

File: tyk-docs/data/menu.yaml [hunk 1]
```diff
@@ -4685,6 +4685,14 @@ menu:
                 path: /tyk-configuration-reference/tyk-identity-broker-configuration
                 category: Page
                 show: True
+          - title: "Release Notes"
+            category: Directory
+            show: True
+            menu:
+              - title: "Tyk Identity Broker 1.6"
+                path: /product-stack/tyk-identity-broker/release-notes/tib-v1.6.md
```
Comment (letzya) [inline tyk-docs/data/menu.yaml]:
```suggestion
                path: /product-stack/tyk-identity-broker/release-notes/tib-v1.6
```

**Review by sredxny — APPROVED** [review]
🚀

**Review by sharadregoti — APPROVED** [review]

**Comment by sharadregoti** [comment]
This PR should be merged together with #5681.

**Pull request merged**